slack_notification/README.md

```diff
@@ -4,26 +4,50 @@ Hook into platform workflows and post notifications to Slack.
+1. [Navigate to api.slack.com/apps](https://api.slack.com/apps) while logged into your Slack workspace.
+1. Click **Create New App**.
+1. Choose **From scratch** in the **Create an app** modal.
+1. Give your app a name (e.g. "Pantheon Deploybot") and select a workspace for your app.
+1. Click **OAuth & Permissions** in the **Features** menu from the left sidebar of your app's configuration screen.
+1. Scroll down to **Bot Token Scopes** under **Scopes** and click the **Add an OAuth Scope** button.
+1. Choose `chat:write` from the dropdown. You may also add other relevant scopes if you plan on extending the Slack notification Quicksilver script's functionality.
+1. Scroll up to **OAuth Tokens** and click the **Install to {your workspace}** button to install the app into your Slack instance.
+1. Authorize ("Allow") the app for your workspace.
+1. Copy the **Bot User OAuth Token** from the **OAuth Tokens** section. We will use [Pantheon Secrets](https://docs.pantheon.io/guides/secrets/overview) to store this token to a secret, bypassing the need for a local file with an API token stored in version control.
+1. You can customize any additional information about your bot, adding an avatar, etc. as you wish.
+
+At this point, you should be able to test the bot manually by sending a `curl` request to the Slack API:
+
+```bash
+curl -X POST -H "Authorization: Bearer xoxb-YOUR-TOKEN" \
+     -H "Content-Type: application/json" \
+     -d '{
+           "channel": "#channel-name",
+           "text": "Hello from Deploybot!"
+         }' \
+     https://slack.com/api/chat.postMessage
 ```
 
-3. Add the example `slack_notification.php` script to the `private` directory in the root of your site's codebase, that is under version control. Note this is a different `private` directory than where the `secrets.json` is stored.
-4. Add Quicksilver operations to your `pantheon.yml`
-5. Test a deployment and see the notification in the Slack channel associated with the webhook.
+### Add the OAuth token to Pantheon Secrets
+
+1. Install the [Terminus Secrets Manager Plugin](https://docs.pantheon.io/guides/secrets#installation).
+1. Set the secret with the following command: `terminus secret:site:set <site> <secret-name> <oauth-token> --scope=web`
+  - Replace `<site>` with your site name (e.g. `my-site`).
+  - Replace `<secret-name>` with the name of your secret that you will use in the code. In the example script, this is set to `slack_deploybot_token`.
+  - Replace `<oauth-token>` with the Bot User OAuth Token copied from the above steps.
+1. Add the example `slack_notification.php` script to the `private` directory in the root of your site's codebase, that is under version control.
+1. Update the script to set the `$slack_channel` (on line 8) to whatever channel you wish to push notifications to.
+1. Update the secret name in the `pantheon_get_secret` function call on line 151 (if you are using a secret that is not named `slack_deploybot_token`).
```

Comment:
similar to my comment on the line above, if there is no good reason to change this string/variable, let's not suggest doing it.